assets/css/theme.css

```diff
@@ -58,6 +58,7 @@ p.highlight {
 a {
   color: #0062af;
   text-decoration: none;
+  border-bottom: 1px solid #767676;
```

Comment:
What's the rationale of using `border-bottom` for text decoration?

Reply:
Well, that is a good question ;)

I confess I did not think about it when I did it. I just reused styles I found elsewhere, and that is how links are styled on W3C's Web site and in W3C specs. Now that you made me think about it, I see two reasons:

1. While `text-decoration` lets one specify a color for the underline, there is no way to set the width of the line. In particular, the goal is to highlight the link on hover through a thicker border, which cannot be done with `text-decoration` (we could use a double line, but the result is so-so)
2. The underline gets drawn closer to the text when `text-decoration` is used, while `border-bottom` leaves more space. For instance, an underline goes through letters such as "g" or "p", while the border does not. I find it slightly more readable when the line gets drawn below the letters.

Both are a matter of taste...

Reply:
OK.

Reply:
* `#1` can be solved by [text-decoration-width](https://drafts.csswg.org/css-text-decor-4/#text-decoration-width-property) (not implemented anywhere for the moment ;)
* I didn't know any good solution for `#2`. `text-underline-position: under;` isn't perfect either...